incubator/hnc/Makefile

```diff
@@ -4,34 +4,49 @@
 # If CONFIG is `kind`, various defaults will be optimized for deploying locally to Kind
 CONFIG ?= "default"
 
-PROJECT_ID=$(shell gcloud config get-value project)
+# The GCP project ID useful to have when performing operations that require one
+# (e.g. release). If you don't have gcloud, all other operations in this
+# makefile should still succeed.
+#
+# We use simple expansion to prevent this from being called every time we need
+# the project ID. gcloud prints some noise to stderr, hence the redirect.
+PROJECT_ID := ${shell gcloud config get-value project 2>/dev/null}
```

Comment:
Question: the purpose of the redirect here is just to not include stderr in the `PROJECT_ID`, right?

Reply:
Exactly. Can I make the comment on line 12 more clear?